include/IMeshBuffer.h

```diff
@@ -863,6 +896,13 @@ GL_INVALID_OPERATION is generated by glVertexAttribPointer if size is GL_BGRA an
             return true;
         }
 
+		//! Sets value of given index of given attribute.
```

Comment:
Big note for this function, NOT ALL FORMAT CONVERSIONS FROM RGBA32F/XYZW32F (vectorSIMDf) ARE IMPLEMENTED!